code/game/gamemodes/gang/gang.dm

```diff
@@ -248,6 +249,19 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 		gang_bosses += G.bosses
 	return gang_bosses
 
+/datum/game_mode/proc/shuttle_check()
+	if(forced_shuttle)
+		return
+	var/alive = 0
+	for(var/mob/living/L in GLOB.player_list)
```

Comment:
@Robustin Why does this populate from `player_list`, which contains non-crew players like the denizens of lavaland, and then compare to `joined_player_list`, which is only the station crew?
Only one list should be used, which of these groups do you want to be using?
In theory as is the round could end immediately if unusually few players join at round-start.